src/cli/devnet_config.hpp

```diff
@@ -0,0 +1,149 @@
+const char *devnet_json = R"foo({
```

Comment:
one last minor thing:

- these configs should be also in taraxa::cli namespace
- #pragma once might be probably used to avoid possible duplicate declaration ?
- maybe it could be even constexpr - not sure about this one if it is possible due to usage ?